src/test/java/test/integration/support/assertions/AnnotationAssert.java

```diff
@@ -23,4 +23,13 @@ public AnnotationAssert withEntry(String entry, String value) {
         Assertions.assertThat(actual.getStringValue(entry)).isEqualTo(value);
         return this;
     }
+
+    public AnnotationAssert withEntries(String... keyValues) {
```

Comment:
instead of this, why not naming it `withPropertyAndValue(String property, String value)`os you can do:

```java
.withPropertyAndValue("containerName", "hello-world").withPropertyAndValue("exposedPort", "8080")
```